reference/3.0/Microsoft.PowerShell.Utility/Select-String.md

```diff
@@ -399,12 +501,11 @@ Accept wildcard characters: False
 
 ### -Path
 
-Specifies the path to the files to be searched.
-Wildcards are permitted.
-The default location is the local directory.
+Specifies the path to the files to search. Wildcards are permitted. The default location is the
+local directory.
 
-Specify files in the directory, such as "log1.txt", "*.doc", or "*.*".
-If you specify only a directory, the command fails.
+Specify files in the directory, such as `log1.txt`, `*.doc`, or `*.*`. If you specify only a
+directory, the command will fail.
```

Comment:
Strike *will*. Passive voice.

If you specify only a directory, the command fails.

Reply:
Thanks. I made the changes and updated the PR.